rotkehlchen/tests/api/test_liquity.py

```diff
@@ -129,3 +129,36 @@ def test_trove_events(rotkehlchen_api_server):
     assert asset == '_ceth_0x6DEA81C8171D0bA574754EF6F8b412F2Ed88c54D'
     assert trove_stake['stake_after']['amount'] == '177.02'
     assert trove_stake['stake_operation'] == 'stake created'
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0xA0446D8804611944F1B527eCD37d7dcbE442caba']])
+@pytest.mark.parametrize('ethereum_modules', [['liquity']])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+def test_account_without_info(
+        rotkehlchen_api_server,
+        start_with_valid_premium,
+        rotki_premium_credentials,
+        inquirer,  # pylint: disable=unused-argument
+):
+    """Test that we can get the status of the trove and the staked lqty"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    # Set module premium is required
+    premium = None
+    if start_with_valid_premium:
+        premium = Premium(rotki_premium_credentials)
```

Comment:
What are you doing here? Why do this?

You want this here:
https://github.com/rotki/rotki/blob/1a3c40b147054eca6b1514feff9d138028ca42e8/rotkehlchen/tests/fixtures/rotkehlchen.py#L161-L164

And don't do it just for liquity. Check all modules of the chain manager (I think they all got premium, right?) and pass its value if it's not None.